tests/test_git_integration.py

```diff
@@ -27,6 +28,31 @@ def test_install(pytester: pytest.Pytester):
     assert re.match(r".*python.* -m nbstripout -t", config['diff "ipynb"']["textconv"])
 
 
+def test_install_different_python(pytester: pytest.Pytester):
+    pytester.run("git", "init")
+    assert pytester.run("nbstripout", "--is-installed").ret == 1
+    pytester.run("nbstripout", "--install", "--python", "DIFFERENTPYTHON")
+    assert pytester.run("nbstripout", "--is-installed").ret == 0
+
+    with open(".git/info/attributes", "r") as f:
+        attr_lines = f.readlines()
+    assert "*.ipynb filter=nbstripout\n" in attr_lines
+    assert "*.zpln filter=nbstripout\n" in attr_lines
+    assert "*.ipynb diff=ipynb\n" in attr_lines
```

Comment:
Suggest dropping this as it's not really relevant to the functionality under test.